Gemfile

```diff
@@ -9,6 +9,4 @@ gemspec
 gem 'bump', require: false
 gem 'rake'
 gem 'rspec'
-gem 'rubocop', github: 'rubocop-hq/rubocop'
-gem 'rubocop-rspec', '~> 1.29.0'
```

Comment:
Also please update this line instead of adding it to the gemspec.

Reply:
Why? What's wrong with `gemspec`?

Reply:
I can't find a reason to change to gemspec. Development dependencies are more convenient to write in Gemfile because develop can add `path` and` github` options.

Reply:
Hm. Gems (sources) can be refine in `Gemfile`, yes, but for what then `development_dependency` in `gemspec`?